packages/components/ng-ovh-utils/src/scroll/scroll-service.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @type service
+ * @name ovhServices:ScrollService
+ * @description
+ * Provide scroll methods
+ * @example
+ * # Usage
+ * <code:js>
+ * export default class PciStoragesContainersContainerController {
+ *  */ /* @ngInject */ /*
+ *  constructor(ScrollService) {
+ *    this.ScrollService = ScrollService
+ *  }
+ *
+ *  $onInit() {
+ *    ScrollService.scrollTop();
+ *  };
+ *};
+ *
+ * </code>
+ */
+export default /* @ngInject */ function scrollService(
+  $document,
+  $timeout,
+  $anchorScroll,
+) {
+  this.scrollTop = function scrollTop() {
+    $document[0].body.style.setProperty('height', 'auto', 'important');
+    $document[0].body.style.setProperty('overflow', 'auto');
+    $anchorScroll();
+    $timeout(function() {
+      $document[0].body.style.removeProperty('height');
+      $document[0].body.style.removeProperty('overflow');
+    });
```

Comment:
**suggestion:** Can we have a better logic to scroll? This seems unusual to me.

Reply:
No we can't because it's not possible to scroll inside an element with no height. And if there is an overflow hidden style on the element, it will give a weird display while scrolling.